README.md

```diff
@@ -6,10 +6,10 @@ A Jekyll plugin to add metadata tags for search engines and social networks to b
 
 ## What it does
 
-Jekyll SEO Tag adds the following meta tags to your site:
+Jekyll SEO Tag adds the following tags to your site:
 
-* Pages title (with site title appended when available)
-* Page description
+* Optional `title` delimited with a pipe
+* Meta description
```

Comment:
There are some things going on here that are outside the scope of this PR.

Meta in this sense is only meaningful to HTML developers. I suspect “Page Description” is more clear to most users.